airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/DateTimeConverter.java

```diff
@@ -158,7 +158,13 @@ public static String convertToTime(final Object time) {
         LOGGER.info("Unknown class for Time data type" + time.getClass());
         loggedUnknownTimeClass = true;
       }
-      return LocalTime.parse(time.toString()).format(TIME_FORMATTER);
+
+      final String valueAsString = time.toString();
+      if (valueAsString.startsWith("24")) {
```

Comment:
do we want handling for more general cases? E.g. if we somehow get `25:00:00`

could do something like... `Integer.parseInteger(valueAsString.substring(0, valueAsString.indexOf(":"))) >= 24`

Reply:
anything above 24 is not allowed in postgres itself. LocalTime only supports till 23:59:59